examples/amp-story/backgrounds.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
```

Comment:
nit: maybe rename this to desktop-backgrounds.html?

Reply:
Done